recipes/dbbot/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "dbbot" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dbbot-{{ version }}.tar.gz
+  sha256: c41fd61b71100eebf757b3b084a104c87dd2b7c15ed7d4166cd9d7540e82e180
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=2.6
+    - pip
+  run:
+    - python >=2.6
```

Metadata has 3.7 there instead.

```suggestion
    - python >=3.7
    - pip
  run:
    - python >=3.7
```

---

Thanks for your answer. 

In the page they say 2.6 https://pypi.org/project/dbbot/. What should it be?

---

We don't build 2.6. We will even stop building 3.7 soon BTW. I'd just leave 3.7 for now.

---

Ok, thank you! 

Updated